src/index.js

```diff
@@ -1 +1,15 @@
 import './style.css';
+import addScore from './modules/submitScore.js';
+import displayScores from './modules/displayScore.js';
+
+// const submitBtn = document.querySelector('.submit-btn');
+const refreshBtn = document.querySelector('.refresh-btn');
+const form = document.querySelector('.form-input');
+
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  addScore();
+  displayScores();
+});
+
+refreshBtn.addEventListener('click', displayScores);
```

Comment:
**OPTIONAL** - To help improve the UX of your page, I suggest that on initial render, your app should display scores from the API. Currently, on initial render, I can only view the scores when I click the refresh button. Please fix this 🙏